nemo_skills/prompt/config/generic/mcq-10choices-aai-boxed.yaml

```diff
@@ -0,0 +1,8 @@
+# Default prompt for multuple choice benchmarks with ten answer choices (e.g. MMLU-Pro). It's intended for models trained to follow \boxed{{}} format and for generaral instruction-following models.
```

Comment:
if prompts are specific for some particular evaluation setup, let's probably move them to the config/eval folder